packages/grafana-ui/src/components/Table/Filter.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useCallback, useMemo, useRef, useState } from 'react';
+import { css, cx } from 'emotion';
+import { Field, GrafanaTheme } from '@grafana/data';
+
+import { TableStyles } from './styles';
+import { stylesFactory, useTheme } from '../../themes';
+import { Icon } from '../Icon/Icon';
+import { FilterPopup } from './FilterPopup';
+import { Popover } from '..';
+
+interface Props {
+  column: any;
+  tableStyles: TableStyles;
+  field?: Field;
+}
+
+export const Filter: FC<Props> = ({ column, field, tableStyles }) => {
+  const ref = useRef<HTMLDivElement>(null);
+  const [isPopoverVisible, setPopoverVisible] = useState<boolean>(false);
+  const theme = useTheme();
+  const styles = getStyles(theme);
```

Comment:
You can use `useStyles` by @tskarhed instead: https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook

Reply:
Yes! I was thinking about creating this myself. Thank you Tobias & Dominik 🌟